releases/release-1.13/release_team.md

```diff
@@ -0,0 +1,13 @@
+| **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
+| ------ | ------ | ------ |
+| Lead | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |
+| Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)), Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / @gsaenger on Slack), Arnaud Meukam ([@ameukam](https://github.com/ameukam)) |
+| CI Signal | Josh Berkus ([@jberkus](https://github.com/jberkus)) | Shivaani Mall ([@shivani1494](https://github.com/shivani1494)), Morten Torkildsen ([@mortent](https://github.com/mortent)) |
```

Comment:
2 more shadows for CI plz

(i) Maria Ntalla (@mariantalla)
(ii) TaoBeier (@tao12345666333)